src/main/java/com/esri/samples/ogc/wfs_xml_query/WfsXmlQuerySample.java

```diff
@@ -0,0 +1,118 @@
+
+    // create stack pane and JavaFX app scene
+    StackPane stackPane = new StackPane();
+    Scene scene = new Scene(stackPane);
+
+    // set title, size, and add JavaFX scene to stage
+    stage.setTitle("Load WFS with XML Query");
+    stage.setWidth(800);
+    stage.setHeight(700);
+    stage.setScene(scene);
+    stage.show();
+
+    // create a progress indicator
+    ProgressIndicator progressIndicator = new ProgressIndicator();
+    progressIndicator.setVisible(true);
+
+    // create an ArcGISMap with topographic basemap and set it to the map view
+    ArcGISMap map = new ArcGISMap(Basemap.createNavigationVector());
+    mapView = new MapView();
+    mapView.setMap(map);
+
+    // define the WFS service URL and layer name
+    String serviceUrl = "https://dservices2.arcgis.com/ZQgQTuoyBrtmoGdP/arcgis/services/Seattle_Downtown_Features/WFSServer?service=wfs&request=getcapabilities";
+    String LayerName = "Seattle_Downtown_Features:Trees";
+    // create a FeatureTable from the WFS service URL and name of the layer
+    WfsFeatureTable wfsFeatureTable = new WfsFeatureTable(serviceUrl, LayerName);
+
+    // set the feature request mode and axis order
+    wfsFeatureTable.setAxisOrder(OgcAxisOrder.NO_SWAP);
+    wfsFeatureTable.setFeatureRequestMode(ServiceFeatureTable.FeatureRequestMode.MANUAL_CACHE);
+
+    // create a feature layer to visualize the WFS features
+    FeatureLayer wfsFeatureLayer = new FeatureLayer(wfsFeatureTable);
+
+    // add the layer to the map's operational layers
+    map.getOperationalLayers().add(wfsFeatureLayer);
+
+    // create an XML query to retrieve trees of genus Tilia.
+    // To learn more about specifying filters in OGC technologies, see https://www.opengeospatial.org/standards/filter.
+    String xmlQuery = "<?xml version=\"1.0\" encoding=\"utf-8\"?>\n" +
+            "<wfs:GetFeature service=\"WFS\" version=\"2.0.0\"\n" +
+            "    xmlns:Seattle_Downtown_Features=\"https://dservices2.arcgis.com/ZQgQTuoyBrtmoGdP/arcgis/services/Seattle_Downtown_Features/WFSServer\"\n" +
+            "    xmlns:wfs=\"http://www.opengis.net/wfs/2.0\"\n" +
+            "    xmlns:fes=\"http://www.opengis.net/fes/2.0\"\n" +
+            "    xmlns:gml=\"http://www.opengis.net/gml/3.2\">\n" +
+            "    <wfs:Query typeNames=\"Seattle_Downtown_Features:Trees\">\n" +
+            "        <fes:Filter>\n" +
+            "            <fes:PropertyIsLike wildCard=\"*\" escapeChar=\"\\\">\n" +
+            "                <fes:ValueReference>Trees:SCIENTIFIC</fes:ValueReference>\n" +
+            "                <fes:Literal>Tilia *</fes:Literal>\n" +
+            "            </fes:PropertyIsLike>\n" +
+            "        </fes:Filter>\n" +
+            "    </wfs:Query>\n" +
+            "</wfs:GetFeature>";
+
+    // populate the WFS feature table with XML query
+    ListenableFuture<FeatureQueryResult> featureQueryResultListenableFuture = wfsFeatureTable.populateFromServiceAsync(xmlQuery, true);
```

Comment:
Don't need a variable for this since you don't use it later. Just call `wfsFeatureTable.populateFromServiceAsync(xmlQuery, true).addDoneListener...`